rpms/s3/s3rpm.spec

```diff
@@ -175,15 +175,13 @@ cd %{_builddir}/%{name}-%{version}-%{_s3_git_ver}/scripts/provisioning/s3setup/s
 python%{py_ver} -m compileall -b *.py
 cp *.pyc %{_builddir}/%{name}-%{version}-%{_s3_git_ver}/scripts/provisioning/s3setup/build/lib/s3setup
 
-echo "build complete"
```

Comment:
I have already addressed this in my PR :)

Reply:
ok, I guess my PR will get merged before yours, so let me do it :)